Source/AssetRipper.Processing/AnimationClips/AnimationClipConverter.cs

```diff
@@ -619,6 +645,35 @@ static void CopyDataToBuffer(IStreamedClip clip, AssetCollection collection, Spa
 			}
 		}
 
+		private float CalculateLastConstantTime(IReadOnlyList<StreamedFrame> streamedFrames)
+		{
+			float stopTime = m_clip.MuscleClip_C74!.StopTime;
```

Comment:
Same as ClipBindingConstant_C74

Reply:
changed `stopTime` to be in the method's argument, and when its called `m_clip.MuscleClip_C74` is not null